vm_groups.go

```diff
@@ -2,14 +2,14 @@ package egoscale
 
 // InstanceGroup represents a group of VM
 type InstanceGroup struct {
-	ID        string `json:"id"`
-	Account   string `json:"account,omitempty"`
-	Created   string `json:"created,omitempty"`
-	Domain    string `json:"domain,omitempty"`
-	DomainID  string `json:"domainid,omitempty"`
-	Name      string `json:"name,omitempty"`
-	Project   string `json:"project,omitempty"`
-	ProjectID string `json:"projectid,omitempty"`
+	Account   string `json:"account,omitempty" doc:"the account owning the instance group"`
+	Created   string `json:"created,omitempty" doc:"time and date the instance group was created"`
+	Domain    string `json:"domain,omitempty" doc:"the domain name of the instance group"`
+	DomainID  string `json:"domainid,omitempty" doc:"the domain ID of the instance group"`
+	ID        string `json:"id,omitempty" doc:"the id of the instance group"`
+	Name      string `json:"name,omitempty" doc:"the name of the instance group"`
+	Project   string `json:"project,omitempty" doc:"the project name of the group"`
+	ProjectID string `json:"projectid,omitempty" doc:"the project id of the group"`
```

Comment:
Can you remove the projectID part

Reply:
Can you? :wink: 

```json
{
  "description": "Creates a vm group",
  "isasync": false,
  "name": "createInstanceGroup",
  "params": [
    {
      "description": "The project of the instance group",
      "length": 255,
      "name": "projectid",
      "related": "",
      "required": false,
      "type": "uuid"
    },
    {
      "description": "the domain ID of account owning the instance group",
      "length": 255,
      "name": "domainid",
      "related": "",
      "required": false,
      "type": "uuid"
    },
    {
      "description": "the account of the instance group. The account parameter must be used with the domainId parameter.",
      "length": 255,
      "name": "account",
      "required": false,
      "type": "string"
    },
    {
      "description": "the name of the instance group",
      "length": 255,
      "name": "name",
      "required": true,
      "type": "string"
    }
  ],
  "related": "",
  "response": [
    {
      "description": "the name of the instance group",
      "name": "name",
      "type": "string"
    },
    {
      "description": "the account owning the instance group",
      "name": "account",
      "type": "string"
    },
    {
      "description": "the project id of the group",
      "name": "projectid",
      "type": "string"
    },
    {
      "description": "time and date the instance group was created",
      "name": "created",
      "type": "date"
    },
    {
      "description": "the domain ID of the instance group",
      "name": "domainid",
      "type": "string"
    },
    {
      "description": "the project name of the group",
      "name": "project",
      "type": "string"
    },
    {
      "description": "the id of the instance group",
      "name": "id",
      "type": "string"
    },
    {
      "description": "the domain name of the instance group",
      "name": "domain",
      "type": "string"
    }
  ]
}
```

Reply:
Of course

Reply:
my current goal is to have the fewer errors reported by `generate/main.go` as possible.